ci/kind/test-e2e-kind.sh

```diff
@@ -44,9 +47,9 @@ YML_CMD=$(dirname $0)"/../../hack/generate-manifest.sh"
 FLOWAGGREGATOR_YML_CMD=$(dirname $0)"/../../hack/generate-manifest-flow-aggregator.sh"
 
 function quit {
-  if [[ $? != 0 ]]; then
-    echoerr " Test failed cleaning testbed"
-    $TESTBED_CMD destroy kind
+  if [[ $? != 0 && $setup_only == "false" ]]; then
```

Comment:
If `--test-only` is set, it seems this would still clean up the cluster when test fails.

Reply:
fixed